action.sh

```diff
@@ -22,8 +22,16 @@ if [[ -z "${HUGO_VERSION}" ]]; then
     echo "No HUGO_VERSION was set, so defaulting to ${HUGO_VERSION}"
 fi
 
-echo "Downloading Hugo: ${HUGO_VERSION}"
-curl -sSL https://github.com/spf13/hugo/releases/download/v${HUGO_VERSION}/hugo_${HUGO_VERSION}_Linux-64bit.tar.gz > /tmp/hugo.tar.gz && tar -f /tmp/hugo.tar.gz -xz
+if [[ "${HUGO_EXTENDED}" = "true" ]]; then
+  EXTENDED_INFO=" (extended)"
+  EXTENDED_URL="_extended"
```

Comment:
This looks to be slightly incorrect to me? The extended URL seems to be:

https://github.com/gohugoio/hugo/releases/download/v0.73.0/hugo_extended_0.73.0_Linux-64bit.tar.gz

So I think the URL needs to be made up like this.

```
URL=https://github.com/spf13/hugo/releases/download/v${HUGO_VERSION}/hugo_${EXTENDED_URL}${HUGO_VERSION}_Linux-64bit.tar.gz
```

and the following change to make it work

```
EXTENDED_URL="extended_"
```

I've tried to do some testing on this, and even when the above changes are working the Hugo binary does not work:

```
bash-5.0# hugo
bash: /usr/local/bin/hugo: No such file or directory
```

Have you encountered this too?

Reply:
Oops, my bad! You are absolutely right, I mixed up the URLs (should be fixed in 11714ab). Regarding the hugo executable: I think you need to use `./hugo` as in `action.sh` instead of just `hugo`. For me, it seems to work. Does that fix it for you as well?

Reply:
Hey.

So the issue seems to be the extended version of Hugo needs glibc, which is not in the alpine docker image. So when you run this in Docker, which is how it runs on GitHub, it cannot execute the binary and you get this error:

```
/usr/bin/action.sh: line 40: /usr/local/bin/hugo: No such file or directory
```

So the options are try and install that library, which doesn't seem to be recommended, or update the container from alpine to something like Debian slim.

Reply:
I've added an extra few commits in readiness for this coming into the project.

https://github.com/benmatselby/hugo-deploy-gh-pages/compare/msl/support_hugo_extended

I don't have enough time this weekend to port from alpine to a Debian based container however.

Reply:
LGTM! Maybe this Docker container could work: https://hub.docker.com/r/frolvlad/alpine-glibc/